assets/src/styles/blocks/PageHeader.scss

```diff
@@ -62,9 +62,9 @@ $text-column-padding-in: 24px;
     padding: 0;
     font-size: inherit;
     font-weight: inherit;
-    line-height: var(--page-header-title--line-height, 1em);
+    line-height: 150%;
     box-decoration-break: clone;
-    box-shadow: 8px 0 0 var(--transparent-button--active--color, #fff), calc(8px * -1) 0 0 var(--transparent-button--active--color, #fff);
```

Comment:
I think the we shouldn't use this `--transparent-button--active--color` since it seems to be related to buttons.

Reply:
Also removing the second color looks like the same,